cmd/kube-scheduler/app/server.go

```diff
@@ -58,8 +58,11 @@ import (
 	"k8s.io/klog"
 )
 
-// NewSchedulerCommand creates a *cobra.Command object with default parameters
-func NewSchedulerCommand() *cobra.Command {
+// Option configures a framework.Registry.
+type Option func(framework.Registry) error
+
+// NewSchedulerCommand creates a *cobra.Command object with default parameters and registryOptions
+func NewSchedulerCommand(registryOptions ...Option) *cobra.Command {
```

Comment:
There is an imported package named `options`, so I use `registryOptions` instead of `options` for the arguments, and it seems more clear for its purpose.